api/v1/menus/getAll.js

```diff
@@ -1,4 +1,9 @@
+const DBClient = require('../../lib/DBClient');
+const dbInstance = new DBClient();
+
 module.exports = (req, res) => {
-    // TODO: getAll menu method.
-    return res.send('TODO: getAll menu method.');
-};
+    dbInstance.get('menus')
+        .then((resp) => res.status(200).json(resp))
+        .catch((err) =>
+            res.status(err.statusCode).json({ ERROR: err.reason }))
```

Comment:
Please, take a look at #15 where I add an error handler to menus, you would only have to call `return next(err)` here, and it would redirect the error to the `menus` error handler.